### PR TITLE
revert node

### PR DESCRIPTION
**Pull request by zikrurR**
### JIRA link

https://tools.hmcts.net/jira/browse/CIV-1827

### Change description

Node needs to be reverted back to 12 as version 14 seems to fail in master but works for PR builds

### Work checklist

- [ ] Unit tests added where applicable
- [ ] Route tests added for new pages
- [ ] New pages included in a11y tests
- [ ] Task list and task completeness checks updated
- [ ] Check and send page updated
- [ ] Routes, page content and page flows of new features are toggled 
- [ ] UI changes look good on mobile
- [ ] Required Google Analytics events are being sent 

### Developer self-QA run statement

- [ ] I have clicked through the running application to see if all changes I made actually work.

### Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -10,7 +10,7 @@ USER hmcts
 
 # ---- Build image ----
 FROM base as build
-RUN PUPPETEER_SKIP_DOWNLOAD=true yarn install
+RUN PUPPETEER_SKIP_CHROMIUM_DOWNLOAD=true yarn install
```
Comment (sabahirfan) [inline Dockerfile:13]: was this change pushed as  part of hmcts/cmc-citizen-frontend#3394?

File: integration-tests.Dockerfile [hunk 2]
```diff
@@ -8,7 +8,7 @@ RUN yarn config set proxy "$http_proxy" && yarn config set https-proxy "$https_p
 
 COPY --chown=hmcts:hmcts package.json yarn.lock /usr/src/app/
 
-RUN PUPPETEER_SKIP_DOWNLOAD=true yarn install && yarn cache clean
+RUN yarn install && yarn cache clean
```
Comment (sabahirfan) [inline integration-tests.Dockerfile:11]: was this change pushed as part of https://github.com/hmcts/cmc-citizen-frontend/pull/3394?

**Comment by sabahirfan** [comment]
When reverting a PR, please prefer to use `Revert` button on bottom of the actual PR we merged, it removes the chances of manual errors.

File: Dockerfile (hunk 1 above)
Comment (zikrurR) [reply Dockerfile:13]: it was part of #3395.

File: integration-tests.Dockerfile (hunk 2 above)
Comment (zikrurR) [reply integration-tests.Dockerfile:11]: it was part of #3395

**Comment by zikrurR** [comment]
> When reverting a PR, please prefer to use `Revert` button on bottom of the actual PR we merged, it removes the chances of manual errors.

I needed to revert 2 PRs.

**Review by mounikahmcts — APPROVED** [review]

**Review by sabahirfan — APPROVED** [review]

**Pull request merged**